spec/lib/solidus_subscriptions/subscription_generator_spec.rb

```diff
@@ -44,6 +44,15 @@
         payment_source: payment_source,
       )
     end
+
+    it 'cleanups the subscription line items fields duplicated on the subscription' do
+      subscription_line_item = create(:subscription_line_item)
```

Comment:
Maybe it's a good idea to add a value to `end_date` here. Just to be sure it's being updated.

Reply:
Good point, I'm going to add also the other 2 fields to highlight the fact that we expect them to change: https://github.com/solidusio-contrib/solidus_subscriptions/pull/202/commits/11fa55f3c6a6a6b95314d1967f354f48b3c17416